include/boost/math/constants/calculate_constants.hpp

```diff
@@ -1006,6 +996,29 @@ inline T constant_two_div_root_pi<T>::compute(BOOST_MATH_EXPLICIT_TEMPLATE_TYPE_
    return 2 * boost::math::constants::one_div_root_pi<T>();
 }
 
+template <class T>
+template<int N>
+inline T constant_feigenbaum<T>::compute(BOOST_MATH_EXPLICIT_TEMPLATE_TYPE_SPEC((boost::integral_constant<int, N>)))
+{
+   // We know the constant to 1018 decimal digits.
+   // See:  http://www.plouffe.fr/simon/constants/feigenbaum.txt
+   // Also: https://oeis.org/A006890
+   // N is in binary digits; so we multiply by log_2(10)
+
+   BOOST_STATIC_ASSERT_MSG(N < 3.321*1018, "\nThe Feigenbaum constant cannot be computed at runtime; it is too expensive. It is known to 1018 decimal digits; you must request less than that.");
+   T alpha{"4.6692016091029906718532038204662016172581855774757686327456513430041343302113147371386897440239480138171659848551898151344086271420279325223124429888908908599449354632367134115324817142199474556443658237932020095610583305754586176522220703854106467494942849814533917262005687556659523398756038256372256480040951071283890611844702775854285419801113440175002428585382498335715522052236087250291678860362674527213399057131606875345083433934446103706309452019115876972432273589838903794946257251289097948986768334611626889116563123474460575179539122045562472807095202198199094558581946136877445617396074115614074243754435499204869180982648652368438702799649017397793425134723808737136211601860128186102056381818354097598477964173900328936171432159878240789776614391395764037760537119096932066998361984288981837003229412030210655743295550388845849737034727532121925706958414074661841981961006129640161487712944415901405467941800198133253378592493365883070459999938375411726563553016862529032210862320550634510679399023341675"};
```

Comment:
Does this still pass in C++03 mode?  I'm not fussed if it only generates an error in C++03 when instantiated, just don't want to break the whole header just yet.

Reply:
Just started a CI cycle to check; the use of `BOOST_STATIC_ASSERT_MSG` was especially to keep C++03 compatibility.